4-WebApp-your-API/4-2-B2C/TodoListService/Startup.cs

```diff
@@ -39,6 +42,15 @@ public void ConfigureServices(IServiceCollection services)
                 });
 
             services.AddControllers();
+            services.AddAuthorization(options =>
+            {
+                // Create policy to check for the scope 'read'
+                options.AddPolicy("ReadScope", 
+                    policy => policy.Requirements.Add(new OperationAuthorizationRequirement { Name = "read" }));
```

Comment:
I'd like to be able to write new OperationScopeRequirement {Scope = "user.read"}